Hairtrigger-Bunnies/nodejs-bangazon-site
# Feature 1- add page verification

Changed:
- added function on page render that checks to see if there is a user
- if yes, takes you to the desired view. if no, takes you to login/register

Because:
- there has to be verification of a user on every page

To test:
1. Run ```npm start```
2. Make sure you are first logged out, then enter ```http://localhost:8080/product/add``` in the url
3. When enter is pressed, since you are not logged in, page should be automatically taken back to login/register page.
4. If logged in, hitting enter with ```http://localhost:8080/product/add``` in the url takes you to the add new product to sell page
